data/api_v1_data_helper.go

```diff
@@ -33,8 +33,8 @@ func getDataV1Params(res *httpResponseWriter) (*apiDataParams, *detailedError) {
 	startDate := query.Get("startDate")
 	endDate := query.Get("endDate")
 	withPumpSettings := query.Get("withPumpSettings") == "true"
-	basalBucket := query.Get("basalBucket") == "true"
-	cbgBucket := (query.Get("cbgBucket") == "true" || query.Get("cbgBucket") == "")
+	basalBucket := query.Get("basalBucket") == "true" || query.Get("basalBucket") == ""
+	cbgBucket := query.Get("cbgBucket") == "true" || query.Get("cbgBucket") == ""
```

Comment:
As all cbg are now in data-read db (and are missing in legacy data.deviceData), we may set this cbgBucket variable to true (or get rid of it).

Reply:
done